xilem/src/view/mod.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
+//! A collection of useful pre-made views (sometimes called 'widgets' in other UI systems)
```

Comment:
It would be useful to have documentation here, but I suspect this is actively misleading. Maybe something like:
```suggestion
//! Views for the widgets which are built-in to Masonry. These are the primitives your Xilem app's view tree will generally be constructed from.
```